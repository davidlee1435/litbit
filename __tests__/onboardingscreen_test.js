// Link.react-test.js
import React from 'react';
import renderer from 'react-test-renderer';
import OnboardingScreen from '../src/OnboardingScreen.js';

test('OnboardingScreen test', () => {
  const tree = renderer.create(<OnboardingScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});