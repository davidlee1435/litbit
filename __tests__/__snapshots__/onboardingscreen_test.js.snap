// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnboardingScreen test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "width": 150,
      }
    }
  >
    <RCTPicker
      items={
        Array [
          Object {
            "label": "East Campus",
            "textColor": undefined,
            "value": "East Campus",
          },
          Object {
            "label": "Ruggles",
            "textColor": undefined,
            "value": "Ruggles",
          },
        ]
      }
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      selectedIndex={0}
      style={
        Array [
          Object {
            "height": 216,
          },
          undefined,
        ]
      }
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={Object {}}
  >
    Room number:
    <TextInput
      onChangeText={[Function]}
      style={
        Object {
          "borderColor": "gray",
          "borderWidth": 1,
          "height": 30,
          "width": 150,
        }
      }
      value=""
    />
  </Text>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
            Object {
              "color": "#841584",
            },
          ]
        }
      >
        submit
      </Text>
    </View>
  </View>
</View>
`;
